controllers/apim/api_controller.go

```diff
@@ -279,7 +338,7 @@ func routeRateLimitsPatch(vHostName, routeName string, actions []*apimv1alpha1.A
 		return nil
 	}
 
-	// A nice trick to make things easy.
+	// A nice trick to make it easier add-in actions
 	stringPatch := string(patchRaw)
 	stringPatch = strings.Replace(stringPatch, "\"ACTIONS\"", string(jsonActions), 1)
 	fmt.Printf("STRING PATCH: %v", stringPatch)
```

Comment:
can remove this

Reply:
But it's such a nice trick! JK, will do...

Reply:
Oh wait, I see, you mean the printf.